### PR TITLE
pd: download bin

### PR DESCRIPTION
**Pull request by HuSharp**

### CODE REVIEW

File: pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy [hunk 1]
```diff
@@ -50,6 +50,25 @@ pipeline {
...
+        stage('Prepare') {
+            steps {
+                dir('pd') {
+                    container("golang") {
+                        sh label: 'pd-server', script: '[ -f bin/pd-server ] || make'
+                        sh label: 'other-server', script: """
+                        chmod +x ${WORKSPACE}/scripts/artifacts/*.sh
+                        ${WORKSPACE}/scripts/artifacts/download_pingcap_artifact.sh --tidb=${REFS.base_ref}
+                        ${WORKSPACE}/scripts/artifacts/download_pingcap_artifact.sh --tikv=${REFS.base_ref}
+                        ${WORKSPACE}/scripts/artifacts/download_pingcap_artifact.sh --tiflash=${REFS.base_ref}
```
Comment (purelind) [inline pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy:62]:
```suggestion
                        ${WORKSPACE}/scripts/artifacts/download_pingcap_artifact.sh --tidb=${REFS.base_ref} --tikv=${REFS.base_ref} --tiflash=${REFS.base_ref}
```

File: pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy [hunk 2]
```diff
@@ -50,6 +50,23 @@ pipeline {
...
+                dir('pd') {
+                    container("golang") {
+                        sh label: 'pd-server', script: '[ -f bin/pd-server ] || make'
+                        sh label: 'other-server', script: """
+                        chmod +x ${WORKSPACE}/scripts/artifacts/*.sh
+                        ${WORKSPACE}/scripts/artifacts/download_pingcap_artifact.sh --tidb=${REFS.base_ref} --tikv=${REFS.base_ref} --tiflash=${REFS.base_ref}
+                        rm -rf third_bin/bin && mv third_bin/* bin/ && ls -alh bin/
+                        bin/pd-server -V
+                        bin/tikv-server -V
+                        bin/tiflash --version
```
Comment (purelind) [inline pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy:64]: How about print tidb version in here
```suggestion
                        bin/tidb-server -V
                        bin/tiflash --version        
```

Comment (HuSharp) [reply pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy:64]: oh... I forgot to print it.. :(

File: pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy [hunk 3]
```diff
@@ -50,6 +50,23 @@ pipeline {
                 }
             }
         }
+        stage('Prepare') {
+            steps {
+                dir('pd') {
+                    container("golang") {
+                        sh label: 'pd-server', script: '[ -f bin/pd-server ] || make'
```
Comment (purelind) [inline pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy:57]: Can the compilation command be modified here to not include the dashboard?

**Review by purelind — APPROVED** [review]
/lgtm

File: pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy (hunk 3 above)
Comment (HuSharp) [reply pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy]: done

**Pull request merged**